ydb/docs/ru/core/reference/configuration/node_authorization.md

```diff
@@ -0,0 +1,49 @@
+# Авторизация узлов при регистрации в кластере
```

Comment:
I think we need add information that client certificates is provided over command line parameters of dynamic node
```
--grpc-ca=/path/to/ca.crt
--grpc-cert=/path/to/client.crt
--grpc-key=/path/to/client.key
```